<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpticResponseMeasureWidget</class>
 <widget class="QWidget" name="OpticResponseMeasureWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ORM scan</string>
  </property>
  <layout class="QVBoxLayout" stretch="2,0,1">
   <item>
    <layout class="QHBoxLayout" stretch="1,1">
     <item>
      <widget class="TableView" name="monitorTable"/>
     </item>
     <item>
      <widget class="TableView" name="opticsTable"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout">
     <item row="3" column="0">
      <widget class="QLabel">
       <property name="text">
        <string>Ignored shots:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="numIgnoredSpinBox">
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel">
       <property name="text">
        <string>Used shots:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="numUsedSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel">
       <property name="text">
        <string>Output directory:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout">
       <item>
        <widget class="QLineEdit" name="folderEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="folderButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <widget class="QLabel">
       <property name="text">
        <string>File name mask:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="fileEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel">
       <property name="text">
        <string>Filter:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="filterEdit">
       <property name="text">
        <string>ax_ | ay_</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel">
       <property name="text">
        <string>Default step [rad]:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="defaultSpinBox">
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="singleStep">
        <double>0.0001</double>
       </property>
       <property name="minimum">
        <number>-1000</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout">
     <item>
      <widget class="QPlainTextEdit" name="logEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <item>
        <widget class="QPushButton" name="startButton">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>madgui/widget/tableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
